### ISSUE TITLE
rewrite rules are not loaded from config file

### ISSUE THREAD

**Issue by carlfish** [lwsjs/local-web-server#62]
node 7.10.0

package.json

```
{
  "scripts": {
     "ws": "ws"
  },
  "devDependencies": {
     "local-web-server": "^1.2.7"
  }
}
```

.local-web-server

```
{
    "rewrite": [
      { "from": "/*", "to": "/dist/$1" }
    ]
}
```

Running "npm run ws" does not load the rewrite rule.

The problem appears to be in cli.js - the rewrite rules returned by the command-line parser are the empty array [], so the Object.assign that merges the config sources overwrites whatever it loads from the config file with the empty list.

Which is weird, because I am _sure_ this worked a month ago.

**Comment by 75lb** [comment]
damn it.. yes, this worked a month ago - this is caused by a regression in command-line-args v4.0.4.

Will fix it this evening, thanks for the report.

**Comment by 75lb** [comment]
as a workaround, running `npm i command-line-args@4.0.3` in whichever folder your local-web-server module is installed should fix it.

**Comment by carlfish** [comment]
Alternatively, downgrading to 1.2.6 seems to work fine. Thanks for the quick response!

**Comment by 75lb** [comment]
the regression was} resolved in command-line-args v4.0.5. Please reinstall local-web-server to pick up the latest deps and fix the issue. 

Let me know if you find anything else! Also, i'm working on lws version 2 - now's a good time for any feature requests. Cheers!
